model/event/host_event_test.go

```diff
@@ -6,18 +6,14 @@ import (
 
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/db"
-	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestLoggingHostEvents(t *testing.T) {
-	Convey("When logging host events", t, func() {
-
-		So(db.Clear(EventCollection), ShouldBeNil)
-
-		Convey("all events logged should be persisted to the database, and"+
-			" fetching them in order should sort by the time they were"+
-			" logged", func() {
+	t.Run("When logging host events", func(t *testing.T) {
```

Comment:
Goconvey unfortunately follows a convention that doesn't really work well with regular non-Goconvey tests, so just replacing a Goconvey test with `testify`/`testing` directly slightly breaks the more modern conventions we have for Go tests. In this case, it would be more conventional to:
1. Remove the outer `t.Run("When logging host events")` because that outer test case isn't actually actually testing/doing anything, and it's pretty clear we're just writing unit tests for `MostRecentHostEvents`.
2. It's more normal to avoid using spaces in the test case name + not using such a long test case name (our [style guide](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.yawpjn5k0xp) recommends PascalCase) because it's a little verbose and Go translates the spaces into underscores when running the test, which show up awkwardly in the parsed test results + makes it slightly confusing to grep for the test. In this case, it would show in the output + Go tests like `all_events_logged_should_be_persisted_to_the_database,_and_fetching_them_in_order...`